zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/monitoring/BackupMXBean.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zookeeper.server.backup.monitoring;
+
+public interface BackupMXBean {
```

Comment:
Let's also make clear what type of metric each one is - for example, is it a gauge? a counter?

Reply:
I added the type of the metric in the comment - The metric names are already pretty long so I think we shouldn't make them longer, and the metric names should tell clearly what this metric does.

Reply:
That should be good enough! Thank you.